roles/core/log_client/tasks/main.yml

```diff
@@ -37,6 +37,20 @@
   tags:
     - template
 
+- name: Allow syslog port into SELinux
+  seport:
```

Comment:
To use `seport`, we need to install some additional packages on the target. You can refer to #394 for examples.
Otherwise, the role fails:
```
TASK [log_client : Allow syslog port into SELinux] *****************************
Sunday 04 October 2020  20:53:04 -0400 (0:00:01.024)       0:00:04.104 ******** 
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'seobject'
failed: [management2] (item=tcp) => changed=false 
  ansible_loop_var: item
  item: tcp
  msg: Failed to import the required Python library (policycoreutils-python) on management2's Python /usr/libexec/platform-python. Please read module documentation and install in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'seobject'
failed: [management2] (item=udp) => changed=false 
  ansible_loop_var: item
  item: udp
  msg: Failed to import the required Python library (policycoreutils-python) on management2's Python /usr/libexec/platform-python. Please read module documentation and install in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter
```

Reply:
on the management1 it seems to work:
```
[root@management1 bluebanquise]# service rsyslog status
Redirecting to /bin/systemctl status rsyslog.service
● rsyslog.service - System Logging Service
   Loaded: loaded (/usr/lib/systemd/system/rsyslog.service; enabled; vendor preset: enabled)
   Active: active (running) since Mon 2020-10-05 15:25:53 CEST; 5min ago
     Docs: man:rsyslogd(8)
           https://www.rsyslog.com/doc/
 Main PID: 76054 (rsyslogd)
    Tasks: 9 (limit: 11479)
   Memory: 1.7M
   CGroup: /system.slice/rsyslog.service
           └─76054 /usr/sbin/rsyslogd -n

Oct 05 15:25:53 management1 systemd[1]: Starting System Logging Service...
Oct 05 15:25:53 management1 rsyslogd[76054]: [origin software="rsyslogd" swVersion="8.1911.0-3.el8" x-pid="76054" x-in>
Oct 05 15:25:53 management1 systemd[1]: Started System Logging Service.
Oct 05 15:25:53 management1 rsyslogd[76054]: imjournal: journal files changed, reloading...  [v8.1911.0-3.el8 try http>
[root@management1 bluebanquise]# netstat -lntp | grep 514
tcp        0      0 0.0.0.0:10514           0.0.0.0:*               LISTEN      76054/rsyslogd
tcp6       0      0 :::10514                :::*                    LISTEN      76054/rsyslogd
[root@management1 bluebanquise]# netstat -lnup | grep 514
udp        0      0 0.0.0.0:10514           0.0.0.0:*                           76054/rsyslogd
udp6       0      0 :::10514                :::*                                76054/rsyslogd
```

and on the computes:
```
[root@management1 bluebanquise]# ssh root@c001
Last login: Mon Oct  5 14:50:23 2020
[root@c001 ~]# service rsyslog status
Redirecting to /bin/systemctl status rsyslog.service
● rsyslog.service - System Logging Service
   Loaded: loaded (/usr/lib/systemd/system/rsyslog.service; enabled; vendor preset: enabled)
   Active: active (running) since Mon 2020-10-05 15:51:56 CEST; 4min 7s ago
     Docs: man:rsyslogd(8)
           https://www.rsyslog.com/doc/
 Main PID: 11410 (rsyslogd)
    Tasks: 3 (limit: 12478)
   Memory: 1.8M
   CGroup: /system.slice/rsyslog.service
           └─11410 /usr/sbin/rsyslogd -n

Oct 05 15:51:56 c001 systemd[1]: Stopped System Logging Service.
Oct 05 15:51:56 c001 systemd[1]: Starting System Logging Service...
Oct 05 15:51:56 c001 rsyslogd[11410]: [origin software="rsyslogd" swVersion="8.1911.0-3.el8" x-pid="11410" x-info="htt>
Oct 05 15:51:56 c001 systemd[1]: Started System Logging Service.
Oct 05 15:51:56 c001 rsyslogd[11410]: imjournal: journal files changed, reloading...  [v8.1911.0-3.el8 try https://www>
```
```
cat /etc/rsyslog.conf
...
# remote host is: name/ip:port, e.g. 192.168.0.1:514, port optional
*.* @@10.10.0.1:10514
...
```

Reply:
Is SELinux enabled on c001? Is it a diskfull install or a diskless setup?

Reply:
```
[root@c001 ~]# sestatus
SELinux status:                 enabled
SELinuxfs mount:                /sys/fs/selinux
SELinux root directory:         /etc/selinux
Loaded policy name:             targeted
Current mode:                   enforcing
Mode from config file:          enforcing
Policy MLS status:              enabled
Policy deny_unknown status:     allowed
Memory protection checking:     actual (secure)
Max kernel policy version:      31
```

it is a diskless installation.

Reply:
You may have run other roles before **time** that installed the requested RPMS.

```
# clush -Lw c[001-002] grep BLUEBANQUISE_IMAGE_NAME /etc/os-release
c001: BLUEBANQUISE_IMAGE_NAME="noselinux"
c002: BLUEBANQUISE_IMAGE_NAME="selinux"
# clush -Lw c[001-002] getenforce
c001: Disabled
c002: Enforcing
# ansible-playbook playbooks/logs.yml --limit c001,c002
[WARNING]: Unable to parse /etc/bluebanquise/internal as an inventory source

PLAY [log server playbook] *************************************************************************************************************************************************************************************
skipping: no hosts matched

PLAY [log clients playbook] ************************************************************************************************************************************************************************************

TASK [Gathering Facts] *****************************************************************************************************************************************************************************************
Monday 05 October 2020  12:18:07 -0400 (0:00:00.045)       0:00:00.045 ********
ok: [c001]
ok: [c002]

TASK [log_client : Gathering OS specific variables] ************************************************************************************************************************************************************
Monday 05 October 2020  12:18:08 -0400 (0:00:01.362)       0:00:01.408 ********

TASK [log_client : Install packages] ***************************************************************************************************************************************************************************
Monday 05 October 2020  12:18:08 -0400 (0:00:00.106)       0:00:01.514 ********
ok: [c001]
ok: [c002]

TASK [log_client : Template >> /etc/rsyslog.conf] **************************************************************************************************************************************************************
Monday 05 October 2020  12:18:10 -0400 (0:00:01.413)       0:00:02.927 ********
ok: [c001]
ok: [c002]

TASK [log_client : Allow syslog port into SELinux] *************************************************************************************************************************************************************
Monday 05 October 2020  12:18:11 -0400 (0:00:00.960)       0:00:03.887 ********
skipping: [c001] => (item=tcp)
skipping: [c001] => (item=udp)
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'seobject'
failed: [c002] (item=tcp) => changed=false
  ansible_loop_var: item
  item: tcp
  msg: Failed to import the required Python library (policycoreutils-python) on c002's Python /usr/libexec/platform-python. Please read module documentation and install in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ModuleNotFoundError: No module named 'seobject'
failed: [c002] (item=udp) => changed=false
  ansible_loop_var: item
  item: udp
  msg: Failed to import the required Python library (policycoreutils-python) on c002's Python /usr/libexec/platform-python. Please read module documentation and install in the appropriate location. If the required library is installed, but Ansible is using the wrong Python interpreter, please consult the documentation on ansible_python_interpreter

TASK [log_client : Manage services state] **********************************************************************************************************************************************************************
Monday 05 October 2020  12:18:11 -0400 (0:00:00.712)       0:00:04.600 ********
ok: [c001] => (item=rsyslog)

PLAY RECAP *****************************************************************************************************************************************************************************************************
c001                       : ok=4    changed=0    unreachable=0    failed=0    skipped=2    rescued=0    ignored=0
c002                       : ok=3    changed=0    unreachable=0    failed=1    skipped=1    rescued=0    ignored=0
```

Your latest commits fixed this condition.

Idea (💡): For diskless, when we enable support for SELinux in the image, we should install python3-policycoreutils.